linkerd/app/outbound/src/http/detect.rs

```diff
@@ -6,7 +6,8 @@ use linkerd_app_core::{
 use tracing::debug_span;
 
 impl<T> Outbound<T> {
-    pub fn push_detect_http<TSvc, H, HSvc, I>(
+    /// Pushes a `DetectHttp` stack with a `SkipByProfile` layer.
```

Comment:
"The `SkipByProfile` layer" is an implementation detail... what does this mean functionally?